tests/check_odov2.sh

```diff
@@ -204,9 +204,9 @@ for stack in $STACKS; do
   devfile_name=$($YQ_PATH eval '.metadata.name' $devfile_path)
   devfile_version=$($YQ_PATH eval '.metadata.version' $devfile_path)
 
-  # Skipping the java-wildfly-bootable-jar stack right now since it's broken.
+  # Skipping the java-wildfly-bootable-jar and java-quarkus stacks right now since they are broken.
   # TODO: Uncomment once fixed.
```

Comment:
Thought this words it better:
```suggestion
  # TODO: Remove condition once fixed.
```